app/views/comments/_comment.html.haml

```diff
@@ -8,10 +8,24 @@
       = link_to user_path(comment.commenter) do
         = comment.commenter.name
       |
-      = link_to 'Reply', new_comment_comment_path( comment )
+      = link_to('Reply', '#', data: { toggle: 'modal', target: "#replyModal_#{comment.id}" })
+      .modal.fade{id: "replyModal_#{comment.id}",'aria-labelledby' => 'replyModalLabel', role: 'dialog', tabindex: '-1'}
```

Comment:
I don't see the point of rendering a `modal` per reply link. Maybe this should be moved to an action in the controller and call it via AJAX :thinking: . What do you think?

Reply:
yeah, this approach will be better 😄 👍

Reply:
actually theres no need of ajax as well i could just change the reply form's action everytime a reply btn is clicked with data attribute for which reply is clicked